docs/builtins.rst

```diff
@@ -0,0 +1,217 @@
+================
+Builtins in L20n
+================
+
+L20n ships with a number of built-in functions that can be used from within
+localization messages.
+
+The list of built-ins is extensible and environments may want to add additional
+built-ins designated to aid localizers writing translations targeted
+for a given environment.
+
+
+How to use Built-ins
+====================
+
+Localizers can reference builtin in a message within a placeable and either
+display the return value, or use it in the selector expression to select
+the right value.
```

Comment:
`FTL Functions can only be called inside of placeables. Use them to return a value to be interpolated in the message or as selectors in select expressions.  Functions may (but don't have to) accept positional and keyword arguments. See the reference below for more information about the arguments accepted for each built-in function.`